src/pretix/control/views/shredder.py

```diff
@@ -103,7 +104,7 @@ def post(self, request, *args, **kwargs):
         if constr:
             return self.error(ShredError(self.get_error_url()))
 
-        if not self.request.user.check_password(request.POST.get("password")):
-            return self.error(ShredError(_("The current password you entered was not correct.")))
+        if request.event.slug != request.POST.get("slug"):
+            return self.error(ShredError(_("The current slug you entered was not correct.")))
```

Comment:
Current slug → event slug? Best take the same error message as with event deletion, this saves translator time :)